include/glow/Importer/Caffe2.h

```diff
@@ -37,26 +36,14 @@ class Tensor;
 class Value;
 
 /// Loads caffe2 models.
-class caffe2ModelLoader {
-  /// The graph that we are constructing.
-  Function &G_;
-  /// Saves network nodes by name.
-  std::unordered_map<std::string, Node *> nodeByName_;
-  /// A list of weight tensors indexed by name.
-  std::unordered_map<std::string, Tensor *> tensors_;
-  /// The external output of the network.
-  SaveNode *root_{nullptr};
-
+class caffe2ModelLoader : public ModelLoader {
```

Comment:
if we do not want polymorphic behavior from caffe2ModelLoader/OnnxLoader and just inherit implementation I'd suggest using private inheritance here.

Reply:
https://onlinegdb.com/S1sKUMbJm

Reply:
We use public methods of base class. (I honestly tried to compile with `private`, but interpreted error messages wrong)

Reply:
"We use public methods of base class" yes, this explains. Thanks for looking. I did not think that's happening.